app/services/analytics.rb

```diff
@@ -148,8 +148,6 @@ def session_started_at
   IDV_INTRO_VISIT = 'IdV: intro visited'
   IDV_JURISDICTION_VISIT = 'IdV: jurisdiction visited'
   IDV_JURISDICTION_FORM = 'IdV: jurisdiction form submitted'
-  IDV_PHONE_CONFIRMATION_FORM = 'IdV: phone confirmation form'
```

Comment:
Looks like this is still referenced on `main` here, causing a 500 for me locally:

https://github.com/18F/identity-idp/blob/bfc2e2274e2458d9bc865a318e0b51de9bdc532c/app/services/funnel/doc_auth/register_step_from_analytics_submit_event.rb#L6

I hit it after entering my password for a newly created account. Surprised that's not covered in a feature spec 🤔

Reply:
> Surprised that's not covered in a feature spec 🤔

Oh, it is 😲 

https://app.circleci.com/pipelines/github/18F/identity-idp/72761/workflows/e4ed12fd-723b-4d2c-8d4e-9d6144db792c/jobs/122933